app/controllers/api_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ApiController < ActionController::API
```

Comment:
Please add some specs around this work. We'd want to make sure that

* encoding and decoding works in the happy path.
* probably want to set an expiry on the token and make sure it expires properly

Reply:
I'm working on getting testing set up for React and will spend a good chunk of time getting myself up to date here.

Reply:
Yes, I'm adding the backend Unit specs as well. Will be doing this next.